src/pool/runner.rs

```diff
@@ -45,6 +52,14 @@ pub trait Runner {
     /// Called when the runner is woken up.
     fn resume(&mut self, _local: &mut Local<Self::TaskCell>) {}
 
+    /// Called when the runner is about to be removed from scheduler.
+    ///
+    /// It's used in the scenario of the scale workers. It will treat this
+    /// thread as a dead thread. It's guaranteed that no other method will
+    /// be called after this method, unless the thread pool is shutdown down,
+    /// in which case end will be the last mtehod called.
```

Comment:
`end` should be quoted.

> unless the thread pool is shutdown down

I think other methods will be called again when being scaled up.

Reply:
Yes, the description of selecting and unselecting during scaling up has been corrected, thank you.